pkg/daemon/ceph/client/image.go

```diff
@@ -100,7 +104,15 @@ func CreateImage(context *clusterd.Context, clusterName, name, poolName, dataPoo
 		}
 	}
 
-	return &CephBlockImage{Name: name, Size: size}, nil
+	// report the adjusted size which will always be >= to the requested size
+	var newSizeBytes uint64
+	if sizeMB > 0 {
+		newSizeBytes = uint64(sizeMB) * BytesPerMB
```

Comment:
Do you mind using this https://github.com/rook/rook/blob/master/pkg/util/display/bytes.go#L68 instead?

Reply:
done